src/components/FilterPanel.jsx

```diff
@@ -34,6 +90,13 @@ export default function FilterPanel({ onClose }) {
   const [grades, setGrades] = useState(params.grades);
   const [majors, setMajors] = useState(params.majors);
 
+  function resetFilters() {
```

Comment:
```suggestion
  function clearAllFilters() {
```

"reset" sounds like the initial state is restored.